tests/test_multiline.py

```diff
@@ -17,7 +17,9 @@ class TestMssqlCliMultiline:
         ('select 1 go', False),
         ('select 1\ngo go go', False),
         ('GO select 1', False),
-        ('GO', True)
+        ('GO', True),
+        ('select 1;select 1\nGO\nselect 1;select 1\nGO', True),
```

Comment:
Can we add named parameters for readability? `is_complete: true `or whatever the syntax is. Also, what is is_complete is a little ambiguous. Perhaps a better name: is_valid_sqlcmd?

Reply:
Sorry for getting to this so late—I changed the method name from `is_complete` to `is_query_executable`. I think this is an improvement because it more clearly designates that the multiline query would be processable. Thoughts?

Reply:
yes, that's much more readable